portlet/web/src/main/webapp/vue-apps/administration-navigation/components/ExoAdministrationHamburgerNavigation.vue

```diff
@@ -46,6 +46,7 @@ export default {
         // data leak prevention
         'dlp-quarantine': 'dataLeakPrevention',
         'transferRules': 'dataLeakPrevention',
+        'multifactor-authentication': 'dataLeakPrevention',
```

Comment:
why is it still named dataLeakPrevention ? Is it possible to rename is security ?